docs/README.md

```diff
@@ -334,11 +334,11 @@ You should see something like this:
 | pgroll                              | postgres          |
 | public                              | pg_database_owner |
 | public_01_create_users_table        | postgres          |
-| public_02_user_description_not_null | postgres          |
+| public_02_user_description_set_nullable | postgres          |
```

Comment:
the preceding lines in the table should be adjusted so that the `|` are aligned.